case-lib/lib.sh

```diff
@@ -339,6 +339,29 @@ find_ldc_file()
     printf '%s' "$ldcFile"
 }
 
+# Prints the syst collateal file found on stdout, logs on stderr.
+find_syst_dict_file()
+{
+    local dict_file
+    # if user doesn't specify file path of sof-*.ldc, fall back to
+    # /etc/sof/sof-PLATFORM.ldc, which is the default path used by CI.
```

Comment:
You copied this comment from the `find_ldc_file()` function that does this fallback but it does not seem to happen here. Which is OK because `/etc/sof` is bad, everything should be in a single place `/lib/firmware/intel/sof`. So just the comment is old and wrong?